File: Modul 1/Soal 10/PRAK110-2610817210004-SoleholAkmal.c
#include <stdio.h>
#include <math.h>

int main() {
    int base = 5;
    int height = 12;
    int hypotenuse = sqrt((base * base) + (height * height));

    int perimeter = height + hypotenuse + base;
    int area = 0.5 * base * height;

    printf("\nDiketahui :");
    printf("\nAlas = %d", base);
    printf("\nTinggi = %d", height);
    printf("\n\nJawab :");
    printf("\nSisi A = %d", height);
    printf("\nSisi B = %d", hypotenuse);
    printf("\nSisi C = %d", base);
    printf("\nKeliling = %d", perimeter);
    printf("\nLuas = %d\n\n", area);

    return 0;
}

File: Modul 1/Soal 10/PRAK110-2610817210004-SoleholAkmal.py
import math

base = 5
height = 12
hypotenuse = int(math.sqrt((base * base) + (height * height)))

perimeter = height + hypotenuse + base
area = int(1/2 * base * height)

print(f"\nDiketahui :")
print(f"Alas = {base}")
print(f"Tinggi = {height}")
print(f"\nJawab :")
print(f"Sisi A = {height}")
print(f"Sisi B = {hypotenuse}")
print(f"Sisi C = {base}")
print(f"Keliling = {perimeter}")
print(f"Luas = {area}\n")

